builder/profitbricks/step_create_server.go

```diff
@@ -22,9 +22,7 @@ func (s *stepCreateServer) Run(ctx context.Context, state multistep.StateBag) mu
 
 	profitbricks.SetAuth(c.PBUsername, c.PBPassword)
 	profitbricks.SetDepth("5")
-	if c.Comm.SSHPublicKey != nil {
-		c.SSHKey = string(c.Comm.SSHPublicKey)
-	} else {
+	if c.Comm.SSHPublicKey == nil {
```

Comment:
I'm happy leaving this now because it's low touch and this isn't worth a refactor, but I don't think this will ever even happen. The builder has a step to create these ssh keys.